doc/ref/configuration/master.rst

```diff
@@ -484,6 +484,21 @@ grains for the master.
 
     enable_gpu_grains: True
 
+.. conf_master:: skip_grains
+
+``skip_grains``
+---------------------
+
+Default: ``False``
+
+MasterMinions should omit grains. A MasterMinion is "a minion function object
+for generic use on the master" that omit pillar. A RunnerClient creates a
+MasterMinion omitting states and renderer. Setting to True can improve master
+performance.
+
+.. code-block:: yaml
+    skip_grains: True
```

Comment:
I think there should be an empty line after `.. code-block:: yaml` - at least docs are complaining about being able to build:
```
10:53:04  Warning, treated as error:
10:53:04  /var/jenkins/workspace/pr-docs_PR-56098/doc/ref/configuration/master.rst:499:Error in "code-block" directive:
10:53:04  maximum 1 argument(s) allowed, 3 supplied.
10:53:04  
10:53:04  .. code-block:: yaml
10:53:04      skip_grains: True
10:53:07  Makefile:56: recipe for target 'html' failed
```

Reply:
Thank you, @waynew. I overlooked the empty line.